### PR TITLE
Removed Scala code from Java docs (#22811)

### PR DESCRIPTION
**Pull request by zaharidichev**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @zaharidichev,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: akka-docs/rst/java/remoting-artery.rst [hunk 1]
```diff
@@ -134,8 +138,8 @@ Looking up Remote Actors
 
 ``actorSelection(path)`` will obtain an ``ActorSelection`` to an Actor on a remote node, e.g.::
 
-  val selection =
-    context.actorSelection("akka://actorSystemName@10.0.0.1:25520/user/actorName")
+  ActorSelection selection =
+    context.actorSelection("akka.tcp://app@10.0.0.1:2552/user/serviceA/worker");
```
Comment (johanandren) [inline akka-docs/rst/java/remoting-artery.rst]: No `tcp` when using artery and I think we tried to keep 25520 as the port in the examples, so please just keep the original path.

**Review by johanandren — COMMENTED** [review]
Thanks, one minor thing to fix.

**Review by johanandren — APPROVED** [review]

**Comment by johanandren** [comment]
Fixes #22811

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Pull request merged**